### ISSUE TITLE
Problème de remontées de région avec offre

### ISSUE THREAD

**Issue by edetaillac** [Evaneos/pyrite#51]
Sur des destis comme `Afrique du Sud`, les régions remontent alors qu'il n'y a pas d'offre.
Ex: https://www.evaneos.fr/afrique-du-sud/

**Comment by RomainJeff** [comment]
@edetaillac en fait pour `offer_public` à partir du moment ou un agent actif est sur la destination, il y a de l'offre... Donc faut juste clarifier ce qu'on entend par "offre" car dans le contexte de `offer-bo` un agent actif = offre disponible.

**Comment by edetaillac** [comment]
hum OK, c'est interessant, le `has_offer` c'est "j'ai un agent actif sur la desti" si je résume ?

**Comment by RomainJeff** [comment]
Yes, en soit c'est pas déconnant si tu considère que tu as un agent actif sur la destination, ça veut dire que tu es capable de gérer des demandes de devis donc en soit tu as de l'offre, mais tu as pas forcément d'itinéraires...
